lib/configurators/databases/databases.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databases
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/lib/config"
+)
+
+// BootstrapFlags flags provided by users to configure and define how the
+// configurators will work.
+type BootstrapFlags struct {
+	// ConfigPath database agent configuration path.
+	ConfigPath string
+	// Manual boolean indicating if the configurator will perform the
+	// instructions or if it will be the user.
+	Manual bool
+	// PolicyName name of the generated policy.
+	PolicyName string
+	// AttachToUser user that the generated policies will be attached to.
+	AttachToUser string
+	// AttachToRole role that the generated policies will be attached to.
+	AttachToRole string
+	// IncludeRedshiftWildcard indicates if Redshift permissions must be
+	// included using a wildcard.
+	IncludeRedshiftWildcard bool
+	// ForceRDSPermissions forces the presence of RDS permissions.
+	ForceRDSPermissions bool
+	// ForceAuroraPermissions forces the presence of Redshift permissions.
+	ForceRedshiftPermissions bool
```

Comment:
Are these needed? Can we simplify and just determine permissions based on the config file alone for now? Would probably simplify a few things.

Reply:
We need them to use the configurator for the `aws create-iam` and `aws print-iam` commands. Those commands define if the policy should have RDS/Redshift permissions using flags (instead of the config file).